cartography/intel/crxcavator/crxcavator.py

```diff
@@ -195,8 +246,31 @@ def load_extensions(extensions, session, update_tag):
     e.lastupdated = {UpdateTag}
     """
 
+    permissions_ingestion_cypher = """
+    UNWIND {Permissions} as permission
+    MERGE (e:ChromeExtensionPermission{name: permission})
```

Comment:
Also, I'm not familiar with this new-style syntax.  `permission` looks like it's an object, but you're making it the `name` member?  Does it work?

I'm out of the loop on this lol.

Reply:
Permission is just a string after parsing it - see details on next comment